src/analyses/variable-sensitivity/abstract_object.cpp

```diff
@@ -206,6 +206,19 @@ void abstract_objectt::output(
   }
 }
 
+abstract_objectt::combine_result abstract_objectt::merge(
+  const abstract_object_pointert &op1,
+  const abstract_object_pointert &op2,
+  const locationt &merge_location,
+  const widen_modet &widen_mode)
+{
+  abstract_object_pointert result = merge(op1, op2, widen_mode).object;
+  result = result->merge_location_context(merge_location);
```

Comment:
Is a separate call better than an extra argument here?

Reply:
Ah; I think I see it.  It is the partner of `write_location_context` which can't easily be folded into eval.